recipes/gau2grid/meta.yaml

```diff
@@ -0,0 +1,142 @@
+      build:
+          ignore_run_exports:
+              # these needed as tools to build and interpreter/module to run, but generic version in both cases
+              - numpy
+              - python
+      requirements:
+          build:
+              - cmake >=3.0
+              - {{ compiler('c') }}
+              - ninja  # [win]
+          host:
+              - numpy
+              - python
+          run:
+              - numpy
+              - python
+      files:
+          - lib/python{{ PY_VER }}/site-packages/gau2grid  # [unix]
+          - Lib/site-packages/gau2grid                     # [win]
+      test:
+          requires:
+              - pytest
+          imports:
+              - gau2grid
+          commands:
+              - ls -l $SP_DIR/gau2grid                          # [unix]
+              - dir %SP_DIR%\gau2grid                           # [win]
+              - export PYLIB_EXT=`$PYTHON -c 'from numpy import distutils; print(distutils.misc_util.get_shared_lib_extension(is_python_ext=False))'`  # [unix]
+              - set PYLIB_EXT=`$PYTHON -c 'from numpy import distutils; print(distutils.misc_util.get_shared_lib_extension(is_python_ext=False))'`     # [win]
+              # Verify library
+              - test -f $SP_DIR/gau2grid/gg$PYLIB_EXT           # [unix]
+              - if not exist %SP_DIR%\\gau2grid\\gg.dll exit 1  # [win]
+              # Inspect linkage
+              - ldd -v $SP_DIR/gau2grid/gg$PYLIB_EXT            # [linux]
+              - otool -L $SP_DIR/gau2grid/gg$PYLIB_EXT          # [osx]
+              - conda inspect linkages --show-files --groupby=dependency pygau2grid  # [unix]
+              - conda inspect objects -p $PREFIX $PKG_NAME      # [osx]
+              # Actually test
+              - export GAU2GRID_FORCE_C_TEST=1                  # [unix]
+              - set GAU2GRID_FORCE_C_TEST=1                     # [win]
+              - python -c "import sys; import gau2grid as gg; sys.exit(gg.test())"
+      about:
+          home: https://github.com/dgasmith/gau2grid
+          dev_url: https://github.com/dgasmith/gau2grid
+          doc_url: https://github.com/dgasmith/gau2grid/blob/master/README.md
+          doc_source_url: https://github.com/dgasmith/gau2grid/blob/master/docs/source/index.rst
+          license: BSD-3-Clause
+          license_url: https://opensource.org/licenses/BSD-3-Clause
+          license_file: LICENSE
+          license_family: BSD
+          summary: "D.G.A. Smith's C-based python module for fast computation of a Gaussian and its derivative on a grid"
+          description: >
+            A collocation code for computing gaussians on a grid of the form:
+            ```
+            out_Lp = x^l y^m z^n \sum_i coeff_i e^(exponent_i * (|center - p|)^2)
+            ```
+            , where the returned matrix dimension are the angular momentum (L) by number of requested points (p).
+
+about:
+  home: https://github.com/dgasmith/gau2grid
```

Comment:
This indentation is 2 spaces, while others are 4 spaces. Can you make it consistent? (Sometimes bot doesn't work with 4 spaces)